Resources/Prototypes/_NF/Entities/Structures/Power/psmes.yml

```
@@ -0,0 +1,77 @@
+- type: entity
+  parent: BaseSMES
+  id: PortableSMES
+  suffix: Portable, 8MJ
+  name: Portable SMES
+  description: A portable SMES. More portable anyways. 
```

```suggestion
  description: A "portable" SMES.  Charges through HV cable, steps down to LV cable.
```
Brief description of how it works?